book/src/exec.md

```diff
@@ -0,0 +1,174 @@
+# Taiga execution model
+
+## On a high level
+**Intents** are a mechanism that allows users to enact a multiparty state transition. Users express their interests with intents, and Taiga makes sure that the intents are satisfied atomically.
+
+Here is the high level description of the process, starting from creating intents and ending up publishing a blockchain transaction representing the desired state transition:
+1. **Create**: A user creates an intent, publishing the information of what they have and what they want
+2. **Gossip**: The intent goes to the intent gossip network and is gossiped around the solver nodes
+3. **Solve**: Solvers find matching intents and update the intent with partial transactions until the intent is fully satisfied
```

Comment:
By "updating", you mean that with the creation of these partial transactions, the intents may (or may not be satisfied? may not) be satisfied, right?

Reply:
Basically it means to add freshly created ptx to the "project". If everyone is happy, then the final tx is created, if not - solvers keep creating and adding ptxs to the "project"